tasks/queue-granules/tests/index.js

```diff
@@ -710,7 +781,7 @@ test.serial('If a childWorkflowMeta is provided, it is passed through to the mes
   );
 });
 
-test('createdAt for queued granule is older than enqueueGranuleIngestMessage date', async (t) => {
+test.serial('createdAt for queued granule is equal to enqueueGranuleIngestMessage date when granules do not have createdAt set', async (t) => { // Todo this can and should be equal
```

Comment:
I'm pretty sure this test wasn't working.   In either case, the premise was incorrect as the record validation is >=

Reply:
2922e830e